### ISSUE TITLE
Duel Suggestions (PLEASE!)

### ISSUE THREAD

**Issue by GodOfBikeRace** [topfreegames/bike-race-issues#28]
Before releasing new things in Bike Race, if I were in you, I would implement duels.
I would change several things there, for example the amount of coins that you receive after a win or after the challenges/free chests. I already suggested this 1 month ago. It can seem superficial, but if you change that, a lot of player would be motivated to play, in order to reach enough coins to level up the bike.

Another thing I would change is the trophy system, I'll upload a pic to understand the idea.
![18928281_10213090055149292_1076172896_n](https://cloud.githubusercontent.com/assets/27891393/26799755/1d957a72-4a37-11e7-984a-f56d9721eeaf.png)
In this way the high levels would be penalized, if they fail with a low level.

More, a lot of people are still complaining about connection problems before and after the race. This is annoying cause it makes you losing a lot of trophies without even playing.

**Comment by MattiePvP** [comment]
I support

**Comment by ccolson4072** [comment]
I support

**Comment by Trolley6969** [comment]
Definitely agree. Plus allow the duel bike you be available to practice on so you can learn the limitations.

**Comment by looneytkp** [comment]
i agree with this

**Comment by zbikSLO** [comment]
I support

**Comment by gyanprakash13** [comment]
yes this deduction is very high, we shouldn't get extra penalized when we're playing with a higher player, it's too easy for a 30 level player to beat a player at level15, so it's not fair
